kpops/component_handlers/topic/handler.py

```diff
@@ -150,7 +150,7 @@ def __check_partition_count(
             log.error(
                 f"Topic Creation: partition count of topic {topic_name} changed! Partitions count of topic {topic_name} is {partition_count}. The given partitions count {topic_spec.partitions_count}."
             )
-            exit(1)  # FIXME raise instead https://github.com/bakdata/kpops/issues/101
+            sys.exit(1)  # FIXME raise instead https://github.com/bakdata/kpops/issues/101
```

Comment:
How about removing the sys.exit(1) and raising an exception instead? As described in the comment/issue

Reply:
Replacing these exit(1)s with exceptions is a better solution, and it is not dependent on the OS.